tests/csrf_tests/tests.py

```diff
@@ -69,6 +69,79 @@ def test_mask_cipher_secret(self):
                 masked = _mask_cipher_secret(secret)
                 self.assertMaskedSecretCorrect(masked, secret)
 
+    def test_get_token_csrf_cookie_set(self):
+        """Test calling when request.META contains "CSRF_COOKIE"."""
+        request = HttpRequest()
+        request.META['CSRF_COOKIE'] = MASKED_TEST_SECRET1
+        self.assertFalse('CSRF_COOKIE_NEEDS_UPDATE' in request.META)
```

Comment:
`self.assertNotIn()` is preferred:
```suggestion
        self.assertNotIn('CSRF_COOKIE_NEEDS_UPDATE', request.META)
```